code/game/objects/items/storage/belt.dm

```diff
@@ -748,6 +748,39 @@
 		/obj/item/ammo_casing/junk,
 	))
 
+/obj/item/storage/belt/bandolier/tactical
+	name = "tactical waist bandolier"
+	desc = "Tactical waist strap bandolier to hold 30 rounds of shotgun-sized ammo."
+	icon_state = "tactibandolier"
+	worn_icon_state = "tactibandolier"
+
+/obj/item/storage/belt/bandolier/tactical/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/update_icon_updates_onmob)
+	atom_storage.max_slots = 30
+	atom_storage.max_total_storage = 30
+	atom_storage.do_rustle = FALSE
+	update_appearance()
+
+/obj/item/storage/belt/bandolier/tactical/update_icon_state() // Looks like an assault belt if empty, otherwise normal icon state
+	if(length(contents))
+		icon_state = initial(icon_state)
+		worn_icon_state = initial(worn_icon_state)
+		return ..()
+	icon_state = /obj/item/storage/belt/military/assault::icon_state
+	worn_icon_state = /obj/item/storage/belt/military/assault::worn_icon_state
+	return ..()
```

Comment:
This is weird, why does it change style when empty

Reply:
so the sprite actually correlates with whether you have any ammo on it unless you want there to be like 30 magic casings on the sprite

Reply:
Why not just make the ammo an overlay

Reply:
true